### PR TITLE
version bump for release

### PR DESCRIPTION
**Pull request by ggabernet**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/airrflow/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/airrflow _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by maxulysse — APPROVED** [review]

File: nextflow.config [hunk 1]
```diff
@@ -254,7 +254,7 @@ manifest {
     description     = 'B and T cell repertoire analysis pipeline with the Immcantation framework.'
     mainScript      = 'main.nf'
     nextflowVersion = '!>=21.10.3'
-    version = '2.3.0dev'
+    version = '2.3.0'
```
Comment (d4straub) [inline nextflow.config:257]: like always, nf-core bump-version doesnt align here nicely. I usually correct that, but sometimes forget it ;)

Comment (ggabernet) [reply nextflow.config:257]: ah good point

Comment (ggabernet) [reply nextflow.config:257]: I think I can't be bothered that all tests need to run again though...

Comment (d4straub) [reply nextflow.config:257]: Wow that one test profile runs >1h 20min, indeed, that alignment isnt worth that :D

**Pull request merged**